### PR TITLE
Add and initialize cucumber

### PR DESCRIPTION
**Pull request by perhegg**
[PT Story: Set up cucumber for acceptance tests](www.pivotaltracker.com/story/show/164490792)
#### Description
Changes proposed in this pull request:

* Add Cucumber-rails
* Add database_cleaner
#### What I have learned working on this feature:
I learnt that I had to add database_cleaner and that it wasn't included in cucumber-rails.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -22,4 +22,6 @@ group :development do
   gem 'listen', '>= 3.0.5', '< 3.2'
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+  gem 'cucumber-rails', require: false
```
Comment (magnus-thor) [inline Gemfile]: These two new gems should be under `development` and `test`

**Review by magnus-thor — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (perhegg) [reply Gemfile]: done!

**Review by magnus-thor — CHANGES_REQUESTED** [review]
Add a `env.rb` for cucumber environment

**Review by magnus-thor — APPROVED** [review]
good job

**Pull request merged**